deployment/eclipse-hawkbit/README.md

```diff
@@ -1,12 +1,13 @@
 <!--
 ******************************************************************************
-Copyright (c) 2019 Bosch Software Innovations GmbH.
+Copyright (c) 2019 Bosch Software Innovations GmbH [and others].
 All rights reserved. This program and the accompanying materials
 are made available under the terms of the Eclipse Public License v2.0
 which accompanies this distribution, and is available at
 https://www.eclipse.org/org/documents/epl-2.0/index.php
 ***************************************************************************** 
 -->
```

Comment:
the same changes to the same file are also part of #82

Reply:
Yup. To avoid duplication I just link to my reply in https://github.com/eclipse/kuksa.cloud/pull/82#discussion_r300247314 .